packages/ui-library/src/theme/variants/button-variants.ts

```diff
@@ -17,6 +17,9 @@ export const buildButtonVariant = (mode: 'light' | 'dark'): Components => ({
       },
       endIcon: {
         marginLeft: '2px',
+        '& > *:first-child': {
+          fontSize: '1.25rem', // 20/16
```

Comment:
what about line height and other props? its gonna affect it

Reply:
This rule just overrides one heavy selector from MUI on endIcons for buttons. By the way, I was able to fix a related issue: In Figma, icons inside buttons are sort of 'absolute' positioned, so the padding starts when the text ends. This was affecting most of our buttons with icons in the app :D 

<img width="507" alt="Screenshot 2024-10-14 at 5 44 03 PM" src="https://github.com/user-attachments/assets/1359f355-74c6-40d6-afa1-a8580f9234d7">